qt/FreeSpaceLabel.cc

```diff
@@ -73,8 +73,7 @@ FreeSpaceLabel::onTimer ()
   tr_variantInitDict (&args, 1);
   tr_variantDictAddStr (&args, TR_KEY_path, myPath.toUtf8 ().constData());
 
-  myTag = mySession->getUniqueTag ();
-  mySession->exec ("free-space", &args, myTag);
+  mySession->exec ("free-space", &args);
```

Comment:
Free space label stopped working, always shows "Calculating Free Space..." now. Take a look at Preferences dialog -> Downloading tab -> Save to Location and Open / Open URL dialogs -> Destination Folder.